src/components/SelectedCounter/SelectedCounter.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SelectedCounter.css';
+
+const SelectedCounter = ({ count, text }) => (
+  <div className="SelectedCounter__container">
+    <span>{`${count} ${text}`}</span>
+  </div>
+);
+
+SelectedCounter.propTypes = {
```

Comment:
We usually declare `propTypes` and `defaultProps` up in a class components as static, or in functional components as `const` declarations:

```
const propTypes = {...}
const defaultProps = {...}

const myReactComponent = ()=> {}

myReactComponent.propTypes = propTypes
myReactComponent.defaultProps = defaultProps